### ISSUE TITLE
automated test set classification

### ISSUE THREAD

**Issue by gngdb** [Neuroglycerin/hail-seizure#9]
Would ideally want a script that will iterate over feature combinations, run train.py to guess which are the best and save output csvs along with information on the exact hyper-parameters used.
